Update Schema Design post with regard to JWT exp (#165)
graphile/graphile.github.io: src/pages/postgraphile/postgresql-schema-design.md

Now, the user would have the permissions of the `forum_example_person` role as they execute their query.

> > **Warning:** Unless explicitly set, JWTs never expire. Once they have been issued they may never be invalidated. This is both good and bad, good in that JWTs are fast in not requiring a database lookup. Bad in that if an attacker gets their hands on a JWT you can’t stop them from using it until the token expires. PostGraphile sets an expiration date of one day to any JWT without an `exp` field.

Reviewer: I've tweaked the wording, and added bold in case you are tempted to only read the first sentence of the warning.

```suggestion
> **Warning:** Unless explicitly set, JWTs never expire. Once they have been issued they may never be invalidated. This is both good and bad, good in that JWTs are fast in not requiring a database lookup. Bad in that if an attacker gets their hands on a JWT you can’t stop them from using it until the token expires. If you do not override `exp` then **PostGraphile's defaults set JWTs to expire after one day**.
```
